etl.yaml

```diff
@@ -5,107 +5,57 @@ metadata:
 
 spec:
   volumes:
-    # To create the crredentials config run --
+    # To create the credentials config run --
     # kubectl create configmap gen3-credentials --from-file Secrets
     - name: gen3-credentials
       configMap:
         name: gen3-credentials
 
-    - name: sheepdog-dbcreds
-      secret:
-        secretName: sheepdog-dbcreds
-
-    - name: useryaml
-      configMap:
-        name: useryaml
-
     - name: studies-mount
+      # {{- if $.Values.global.dev }}          
```

Comment:
Should we add parameters section in global ie etl

Values.global.dev and Values.global.etl.studies_path

Reply:
Tracking ETL Helm Chart PR #9